debian/rules

```diff
@@ -129,6 +129,8 @@ override_dh_shlibdeps:
 override_dh_auto_clean:
 	@# stop debhelper from doing "make clean"
 
+### sergiodj: I tried invoking dh_dwz passing --no-dwz-multifile to
+### see if it could make any difference, to no avail.  Oh, well...
```

Comment:
I tried the same :) do you think we should improve the comment message here?

Reply:
Yes, that's a good idea.